### PR TITLE
Webp encoding

### PR DESCRIPTION
**Pull request by vhuhtine**
- Adds experimental native webp encoding support

### CODE REVIEW

File: transform.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 	"log"
 	"math"
 
+	webpencode "git.sr.ht/~jackmordaunt/go-libwebp/webp"
```
Comment (mahe-work) [inline transform.go]: Why named import? Other "registered" formats are referref by the image format (tiff.Encode(...) etc.) Would it clash with the underscore import of the x/image/webp below (and why is that needed if we have this instead)? :thinking:

Comment (mahe-work) [inline transform.go]: Why do we need this named import (and the underscore import of the x/image/webp below, I guess these are related)?

**Review by mahe-work — APPROVED** [review]

File: transform.go (hunk 1 above)
Comment (vhuhtine) [reply transform.go]: It seems that the names don't clash when the golang.org/x/image/webp is imported with _. 
I think I was just trying to make it very clear what is the experimental part here.

Comment (vhuhtine) [reply transform.go]: All the formats needs to be registered to make image.Decode know them.
